packages/storybook/stories/Select.stories.jsx

```diff
@@ -67,3 +67,29 @@ WithAnalytics.args = {
   ...defaultArgs,
   enableAnalytics: true,
 };
+
+const I18nTemplate = args => {
+  const [lang, setLang] = useState('en');
+  const [value, setValue] = useState(args.value);
+  const onValueChange = newValue => {
+    setValue(newValue);
+  };
+
+  useEffect(() => {
+    document.querySelector('main').setAttribute('lang', lang);
+  }, [lang]);
+
+  return (
+    <div>
+      <button onClick={e => setLang('es')}>Español</button>
+      <button onClick={e => setLang('en')}>English</button>
+      <Select {...args} value={value} onValueChange={onValueChange} />
+    </div>
+  );
+};
+
+export const Internationalization = I18nTemplate.bind({});
+Internationalization.args = {
```

Comment:
This may be happening due to the `Select` that is being imported. I haven't found a way for us to have the prop docs _and_ the internationalization functionality for the React components.



!["undefined" value in UI](https://user-images.githubusercontent.com/2008881/167505607-1fc3359d-a212-48dd-a83d-a8814172a226.png)